libkbfs/disk_block_cache_test.go

```diff
@@ -758,3 +760,51 @@ func TestDiskBlockCacheMoveBlock(t *testing.T) {
 	require.Equal(t, 1, cache.syncCache.numBlocks)
 	require.Equal(t, TriggeredPrefetch, prefetchStatus)
 }
+
+func TestDiskBlockCacheHomeDirPriorities(t *testing.T) {
+	t.Parallel()
+	t.Log("Test that blocks from a home directory aren't evicted when there" +
+		" are other options.")
+	cache, config := initDiskBlockCacheTest(t)
+	defer shutdownDiskBlockCacheTest(cache)
+
+	ctx := context.Background()
+
+	t.Log("Set home directories on the cache")
+	homeTLF := tlf.FakeID(11, tlf.Private)
+	cache.AddHomeTLF(ctx, homeTLF)
+	homePublicTLF := tlf.FakeID(11, tlf.Public)
+	cache.AddHomeTLF(ctx, homePublicTLF)
+
+	t.Log("Seed the cache with blocks")
+	numTlfs := 12
+	numBlocksPerTlf := 5
+	numBlocks := numTlfs * numBlocksPerTlf * 2
```

Comment:
I don't follow this--why is this number true?

Reply:
Each of the two seed calls adds `numTlfs * numBlocksPerTlf` blocks.